Make.inc

```diff
@@ -73,6 +73,9 @@ REAL_ARCH := $(ARCH)
 ifeq ($(findstring arm,$(ARCH)),arm)
 override ARCH := arm
 endif
+ifeq ($(findstring powerpc,$(ARCH)),powerpc)
```

Comment:
I would use `findstring` here too, and check for `ppc` in addition to `powerpc` (on Fedora/RHEL, the arch is called ppc64 and ppc64le).